src/libs/actions/Report.js

```diff
@@ -50,6 +51,18 @@ Onyx.connect({
     },
 });
 
+const allReportActions = {};
+Onyx.connect({
+    key: ONYXKEYS.COLLECTION.REPORT_ACTIONS,
+    callback: (actions, key) => {
+        if (!key || !actions) {
+            return;
+        }
+        const reportID = CollectionUtils.extractCollectionItemID(key);
+        allReportActions[reportID] = actions;
+    },
+});
```

Comment:
This seems to be too much work / data. It's only used once (in deleteReport), can we move the logic there? To make a onyx connection just once and disconnect once data is gathered.

Reply:
Done